set.mm

```diff
@@ -510554,6 +510819,1209 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
   $}
 
 
+$(
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+  Localization of rings
+-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
+$)
+
+  $c ~RL RLocal $.
+
+  $( Syntax for ring localization equivalence class operation. $)
+  cerl $a class ~RL $.
+
+  $( Syntax for ring localization operation. $)
+  crloc $a class RLocal $.
+
+  ${
+    $d r s x w a b t $.
+    $( Define the operation giving the equivalence relation used in the
```

Comment:
It's pretty hard to read condensed formulae, so maybe it's better to make the comment more precise. For example:
```
Define the equivalence relation used to construct the localization of a ring ` r ` by a set ` s `.
Two pairs ` a = <. x , y >. ` and ` b = <. z , w >. ` are equivalent, if there exists ` t e. s `
such that  ` t x. ( x x. w - z x. y ) = 0 `.
This corresponds to the usual comparison of fractions ` x / y ` and ` z / w `.
```

Reply:
Fixed in https://github.com/metamath/set.mm/pull/4810/commits/f635662cfffd84458584f2c4a905b88f32b45a51